usecases/backup/backend.go

```diff
@@ -165,6 +165,8 @@ func (u *uploader) all(ctx context.Context, classes []string, desc *backup.Backu
 	desc.Status = string(backup.Transferring)
 	ch := u.sourcer.BackupDescriptors(ctx, desc.ID, classes)
 	defer func() {
+		//  make sure context is not cancelled when uploading metadata
+		ctx := context.Background()
```

Comment:
Consider using a different variable name for clarity (i.e., instead of shadowing).

Reply:
Shadowing in this case is intended because we want to express that the Clojure has no use for the original context